action.yml

```diff
@@ -6,6 +6,11 @@ description: >
   Publish diff coverage report as PR comment, and create a coverage badge
   to display on the readme.
 inputs:
+  GITHUB_URL:
+    description: >
+      A Github base url, api.github.com by default but can be different in GitHub Enterprise
+    default: "https://api.github.com"
+    required: false
```

Comment:
Actually, it's already injected by GHA as `GITHUB_API_URL`. You don't need to add it in the inputs.
In settings.py, you can read it from the environment directly.

(Also, please make sure you wanted to contribute here and not to https://github.com/py-cov-action/python-coverage-comment-action, people have made the mistake in the past)